src/web/crate_details.rs

```diff
@@ -352,13 +389,19 @@ pub(crate) async fn crate_details_handler(
 
     let details = spawn_blocking(move || {
         let mut conn = pool.get()?;
-        CrateDetails::new(
+        let mut details = CrateDetails::new(
             &mut *conn,
             &params.name,
             &version,
             &version_or_latest,
             Some(&repository_stats_updater),
-        )
+        )?;
+        if let Some(ref mut details) = details {
+            if let Err(e) = details.enrich_readme(&storage) {
+                tracing::debug!("{e:?}")
```

Comment:
what are the errors you are envisioning here? 

Why not just fetch the readme and return errors? 

Or is this just a (kind of) slow rollout?

Reply:
The main one I see happening is the source archive being missing, I believe there was a period where we didn't save it for some kinds of build failures. Other than that, I don't want to fail if theres other things I've missed.

What I should probably do is used debug for just the cases I expect and report other errors to sentry so we can fix them.

Reply:
Changed this to only ignore the readme being missing, all other errors will get reported to sentry.